src/site/content/en/blog/datatransfer/index.md

```diff
@@ -0,0 +1,277 @@
+also limits the user to transfer data to applications that aren't using your format. If you want to
+connect with third-party applications across the web, you need a universal data format.
+
+The [JSON-LD](https://json-ld.org/) (Linked Data) standard is a great candidate for this. It is
+lightweight and easy to read and write to from JavaScript. [Schema.org](Schema.org) contains many
+predefined types that can be used, while custom schema definitions are an option as well.
+
+```js
+const data = {
+  '@context': 'https://schema.org',
+  '@type': 'ImageObject',
+  contentLocation: 'Venice, Italy',
+  contentUrl: 'venice.jpg',
+  datePublished: '2010-08-08',
+  description: 'I took this picture during our honey moon.',
+  name: 'Canal in Venice',
+};
+event.dataTransfer.setData('application/ld+json', JSON.stringify(data));
+```
+
+When using the Schema.org types, you can start with the generic [Thing](schema.org/Thing) type, but
+go down to types like [Event](schema.org/Event), [Person](schema.org/Person),
+[MediaObject](schema.org/MediaObject), [Place](schema.org/Place), and even
+[MedicalEntity](schema.org/MedicalEntity) if you like. If you use TypeScript, you can use the
+interface definitions from the [schema-dts](github.com/google/schema-dts) type definitions.
+
+By transmitting and receiving JSON-LD data, you will support a more connected and open web. With
+applications speaking the same language, you can create deep integrations with external
+applications. No need for complicated API integrations; all the information that's needed is
+included in the transferred data.
+
+Think of all the possibilities when you can transfer data between any (web) application with no
+restrictions. Sharing events from a calendar to your favorite ToDo app, attaching virtual files to
+emails, sharing contacts. That would be great, right? This starts with you! 🙌
+
+## Concerns
+
+While transferring data between web applications with DataTransfer is available and ready to use
+today, there are some things to be aware of before integrating.
+
+### Browser compatibility
+
+Desktop browsers all have great support for the technique described above, while mobile devices have
+not. The technique has been tested on all major browsers (Chrome, Edge, Firefox, Safari) and
+operating systems (Android, Chrome OS, iOS, macOS, Ubuntu Linux, and Windows), but unfortunately Android
+and iOS didn't pass the test. While browsers continue to develop, for now the technique is limited
+to desktop browsers only.
+
+### Discoverability
+
+Drag-drop and copy-paste are system-level interactions when working on a desktop computer, with
+roots back to the first GUIs about 40 years ago. Think for example about how many times you have
+used these interactions for organizing files. On the web, this is not very common yet.
+
+You will need to educate users about this new interaction, and come up with UX patterns to make this
+recognizable.
```

Comment:
I've heard that there are even young people who's experience with computers is confined to mobile devices. Do you need to add some language starting with, "That many mean more than telling them it's like desktop."

Reply:
Added as a by-phrase.